projects/bitcoin-core/project.yaml

```diff
@@ -0,0 +1,20 @@
+homepage: "https://github.com/bitcoin/bitcoin"
+main_repo: 'https://github.com/bitcoin/bitcoin.git'
+language: c++
+primary_contact: "marco@chaincode.com"
+auto_ccs:
+  - "fanquake@gmail.com"
+  - "john@brink.dev"
+  - "jonas@chaincode.com"
+  - "laanwj@gmail.com"
+  - "pieter@chaincode.com"
+  - "thomas.j.bitcoin@protonmail.com"
+sanitizers:
+  - address
```

Comment:
are we passing on memory?

Reply:
The goal here is to add a minimal working example. I'll be working on everything else as a follow up. Or anyone else who beats me to it. Msan needs some fine tuning, but the initial diff is:

```
diff --git a/projects/bitcoin-core/build.sh b/projects/bitcoin-core/build.sh
index ad1a78e8..9592391a 100755
--- a/projects/bitcoin-core/build.sh
+++ b/projects/bitcoin-core/build.sh
@@ -29,7 +29,7 @@ else
 fi
 (
   cd depends
-  make HOST=$BUILD_TRIPLET DEBUG=1 NO_QT=1 NO_WALLET=1 NO_ZMQ=1 NO_UPNP=1 NO_NATPMP=1 -j$(nproc)
+  make HOST=$BUILD_TRIPLET DEBUG=1 NO_QT=1 NO_WALLET=1 NO_ZMQ=1 NO_UPNP=1 NO_NATPMP=1 boost_cxxflags="-std=c++17 -fvisibility=hidden -fPIC ${CXXFLAGS}" zeromq_cxxflags="-std=c++17 ${CXXFLAGS}" -j$(nproc)
 )
 
 ./autogen.sh